src/collective/cover/tiles/list.py

```diff
@@ -222,8 +223,13 @@ def populate_with_uids(self, uuids):
         data_mgr.set(old_data)
         notify(ObjectModifiedEvent(self))
 
-    # XXX: This should be renamed to replace_with_uuids
     def replace_with_objects(self, uuids):
+        warn('Method replace_with_objects() is deprecated. Use ' +
+             'with replace_with_uuids() method instead.',
+             DeprecationWarning, stacklevel=2)
+        self.replace_with_uuids(uuids)
```

Comment:
use this example of deprecation https://github.com/plone/plone.app.layout/blob/master/plone/app/layout/viewlets/common.py#L46-L47